includes/Registry.php

```diff
@@ -48,7 +46,7 @@ public function has( $name ) {
 	/**
 	 * Registers a feature with the registry.
 	 *
-	 * @param mixed $theclass The feature class.
+	 * @param string $theclass The feature class.
 	 */
 	public function set( $theclass ) {
```

Comment:
```suggestion
	public function set( $className ) {
```
More explicit naming